server/routes/request.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const router = express.Router();
+const protect = require("../middleware/auth");
+const { getRequests, postRequest, updateRequest } = require("../controllers/request");
+
+router.route("/").get(protect, getRequests);
+router.route("/new").post(protect, postRequest);
+router.route("/update").put(protect, updateRequest);
```

Comment:
We need to add the aforementioned route param here - we can use "/" for this route as well.

Also, `put` method is counter-intuitive since it implies replacement of a resource as a whole. Let's use `post` or `patch` for this route.